site/assets/css/header.scss

```diff
@@ -10,6 +10,49 @@
   }
 }
 
+.nav-left {
+  overflow: visible;
+  overflow-x: visible;
+  overflow-y: visible;
+  white-space: nowrap;
+}
+
+#searchBoxContainer {
+  width: 220px;
+  height: 36px;
+  top: 50%;
+  margin-top: -18px;
+  margin-left: 16px;
+  white-space: nowrap;
+  position: relative;
+}
+
+#searchBoxContainer .icon{
+  position: absolute;
+  margin-left: 7px;
+  margin-top: 7px;
+  z-index: 1;
+  color: #6f7c76;
+}
+
+#docSearchTextbox {
+  width: 220px;
+  height: 36px;
+  border: none;
+  font-size: 12pt;
+  background-color: #fafafa;
+  float: left;
+  padding-left: 40px;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  border-radius: 5px;
+  outline: none;
+}
```

Comment:
I'm not happy that I needed to add so many styles, so tell if you have any suggestions how to make it simpler.